atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -54,46 +47,59 @@ class SendMailPlugin(ExitPlugin):
     AUTO_SUCCESS = 'auto_success'
     AUTO_FAIL = 'auto_fail'
     AUTO_CANCELED = 'auto_canceled'
+    DEFAULT_SUBMITTER = 'Unknown'
```

Comment:
Its being used in the body later on: `Submitted by %(user)`

Currently it would show either `<autorebuild>` or `Unknown`, which is incorrect, as data fetched from Koji is not being set there.

I'll update `_get_receivers_list` to make sure `self.submitter` is being reset to the user who submitted the koji build, but I think the `__init__` field should be renamed to smth like `cc` or `always_notify`. Thoughts?